release-notes-2.0.0-SNAPSHOT.md

```diff
@@ -0,0 +1,13 @@
+!not-ready-for-release!
```

Comment:
- Remove the `-SNAPSHOT` from the version number in the release notes.  (I understand why you added it but it's not what the build process expects.)